pages/1_14/items/basic_items.md

```diff
@@ -0,0 +1,111 @@
+# Basic Items
+
+At this point you should be able start with creating some basic items.
+
+Let's start with a `ModItems` class.
+
+```java
+package com.example.examplemod.items;
+
+import com.example.examplemod.Reference;
+import net.minecraft.item.Item;
+import net.minecraft.item.Item.Properties;
+import net.minecraftforge.event.RegistryEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.registries.IForgeRegistry;
+import net.minecraftforge.registries.ObjectHolder;
+
+@Mod.EventBusSubscriber(bus=Mod.EventBusSubscriber.Bus.MOD)
+@ObjectHolder(Reference.MODID)
+public class ModItems {
+
+    @ObjectHolder("gem")
+    public static Item GEM = null;
```

Comment:
```suggestion
    public static Item GEM;
```
Java will automatically set it to `null` if you don't specify a value.

Reply:
I think if you don't initialise it the compiler will complain that the field may not be initialised yet.
The field should probably be final tbh.

Reply:
^^ altho yeah i should have made that final.